requirements-dev.txt

```
@@ -2,6 +2,7 @@ build==1.0.3
 coverage==7.4.0
 mypy==1.8.0
 pytest==7.4.4
+responses>=0.25
```

```suggestion
responses==0.25
```

nit; I've pinned other dependencies to avoid unexpected changes to behavior.